src/EditorFeatures/VisualBasicTest/EndConstructGeneration/EndConstructCommandHandlerTests.vb

```diff
@@ -33,7 +33,7 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.EndConstructGenera
         End Sub
 
         <WpfFact, Trait(Traits.Feature, Traits.Features.EndConstructGeneration)>
-        Public Sub ServiceCompletingShouldCallNextHandler()
+public async function ServiceCompletingShouldCallNextHandler()
```

Comment:
Looks like C# made it into your VB code.

Reply:
Fixed.